mkosi/resources/mkosi.md

```diff
@@ -1384,6 +1384,42 @@ local directory:
   output to stdout will be used as the credential value. Output to stderr will be ignored.
   Credentials configured with `Credentials=` take precedence over files in `mkosi.credentials`.
 
+* The **`mkosi.repart/`** directory is used as the source for
+  systemd-repart partition definition files which are passed to
+  systemd-repart when building a disk image. If it does not exist and
+  the `RepartDirectories=` setting is not configured, mkosi will default
+  to the following partition definition files:
+
+  `00-esp.conf` (if we're building a bootable image):
+
+  ```
+  [Partition]
+  Type=esp
+  Format=vfat
+  CopyFiles=/efi:/
+  SizeMinBytes=512M
+  SizeMaxBytes=512M
+  ```
+
+  `05-bios.conf` (if we're building a BIOS bootable image):
+
+  ```
+  [Partition]
+  Type=21686148-6449-6e6f-744e-656564454649
```

Comment:
I think it would be a good idea to put a comment here that this is needed for GRUB to find it

Reply:
It is noted in the section on `BiosBootloader=`, though reminding it here might be helpful.